src/data_vis.ts

```diff
@@ -124,51 +116,43 @@ class DataVisModel {
     filteredTSTable: KnockoutObservableArray<any>;
 
     constructor() {
-        this.currentPlot.subscribe(function() {
-            if (this.currentPlot() == "Impact") {
-                this.renderImpact();
-            } else if (this.currentPlot() == "Time series") {
-                this.renderTimeSeries();
-            }
-        }, this)
-        this.compare.subscribe(function() {
-            this.onUIChange(true, true);
-        }, this)
-        this.disaggregateBy.subscribe(function() {
-            this.onUIChange(true, true);
```

Comment:
I *think* `disaggregateBy` and `cumulativePlot` won't affect the x-axis right? So we don't need to update the options here.